types/react-native/globals.d.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * This file is necessary to declare global functions that might also be included by `--lib dom`.
+ * Due to a TypeScript bug, these cannot be placed inside a `declare global` block in index.d.ts.
+ */
```

Comment:
Can you add the link to the ticket so the reader knows where to find more info and that we can easily keep an eye on it?

Reply:
Yes, I'll update and do a new PR